### PR TITLE
Choose account to merge calendars

### PR DESCRIPTION
**Pull request by barbarasimandi**

### CODE REVIEW

File: app/src/main/res/layout/account_name_with_button.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.83"
+        app:layout_constraintStart_toEndOf="@+id/accountname"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.149" />
+
+</android.support.constraint.ConstraintLayout>
```
Comment (adamgyulavari) [inline app/src/main/res/layout/account_name_with_button.xml]: @barbarasimandi new line!!

File: app/src/main/res/layout/account_name_with_button.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
```
Comment (adamgyulavari) [inline app/src/main/res/layout/account_name_with_button.xml]: @barbarasimandi this should be wrap_content

File: app/src/main/res/layout/activity_choose_account.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="10dp"
+    tools:context="com.greenfox.kalendaryo.ChooseAccountActivity">
+
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/accountNames"
+        android:scrollbars="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```
Comment (adamgyulavari) [inline app/src/main/res/layout/activity_choose_account.xml:23]: @barbarasimandi match_parent

File: app/src/main/java/com/greenfox/kalendaryo/ChooseAccountActivity.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    public List<KalAuth> getAccounts() {
+         ArrayList<String> accountNameList = kalpref.getAccounts();
+         ArrayList<KalAuth> auths = new ArrayList<>();
+
+         for (int i = 0; i < accountNameList.size(); i++) {
+            KalAuth auth = kalpref.getAuth(accountNameList.get(i));
+            auths.add(auth);
+        }
+        return auths;
+    }
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/ChooseAccountActivity.java]: @barbarasimandi lets move this method to `KalPref`

File: app/src/main/res/layout/activity_choose_account.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        android:layout_marginTop="16dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"/>
+
```
Comment (adamgyulavari) [inline app/src/main/res/layout/activity_choose_account.xml]: @barbarasimandi how about a nice and shiny `next` button here?

**Review by adamgyulavari — DISMISSED** [review]

**Review by dzlica — APPROVED** [review]
cool!

**Pull request merged**